docs/thanks.html

```diff
@@ -9,17 +9,20 @@
   <link rel="stylesheet" href="main.css"></link>
 </head>
 <body>
+  <!-- htmllint attr-bans="false" attr-name-style="false" -->
   <svg id="logo" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 100 100">
     <circle cx="50%" cy="50%" r="50%" fill="rgba(82, 14, 33, 0.2)" />
     <use href="firefox-logo.svg#Firefox_-_Large" x="5%" y="5%" width="90%" height="90%" />
   </svg>
+  <!-- htmllint attr-bans="$previous" attr-name-style="$previous" -->
+
   <h1>Thank you!</h1>
   <p>
     Firefox would like to make sure all websites work well for all users. Your feedback helps us<br/>
     understand how we are doing, and if there are any issues we need to fix or report to the site.
   </p>
   <p>
-    You can report sites that aren't working well by clicking on the <img src="broken_page.svg" /> icon.<br/>
+    You can report sites that aren't working well by clicking on the <img src="broken_page.svg" alt="Blipz broken page" /> icon.<br/>
```

Comment:
Not sure how great this `alt=` is for a11y, but it almost reads OK as a sentence.
It'd also be nice to maybe see if we can run these standalone HTML documents against some sort of a11y linter. Only one I can think of is the React JSX a11y linter for ESLint. But maybe I can just run these through a DevTools a11y check manually.

Reply:
Yeah, that's probably not a great alt. Probably something like "broken page icon" (because wtf is a blipz, amirite)